gossip/src/duplicate_shred_handler.rs

```diff
@@ -131,6 +155,18 @@ impl DuplicateShredHandler {
                     shred1.into_payload(),
                     shred2.into_payload(),
                 )?;
+                // feature_epoch could only be 0 in tests and new cluster setup.
+                if self
+                    .enable_gossip_duplicate_proof_ingestion_epoch
+                    .is_some_and(|feature_epoch| {
+                        self.epoch_schedule.get_epoch(slot) > feature_epoch || feature_epoch == 0
```

Comment:
i believe the intention was to avoid having to add a 1 epoch delay to the various local cluster tests that send a duplicate block.
Since this is temporary code it's probably fine to keep it, however if you feel strongly I can remove this and add the delay to relevant tests.

Reply:
Why do you say "i believe the intention was to"?
didn't you add this code?

Reply:
i only added the initial change, wen added the feature flag stuff

Reply:
@wen-coding why do we need `|| feature_epoch == 0`?

I don't see `|| feature_epoch == 0` hack anywhere else in the code-base.

Reply:
Hi, sorry I just noticed this. This was added for the local-cluster tests. Because currently we only enable the feature at the next Epoch, it's either this or enable the feature at next Epoch. I tried to make the local-cluster wait for one Epoch but somehow it got stuck.

Is there a better way to force enable the feature in local-cluster tests?

Reply:
@wen-coding putting hacks into the code just so that the tests pass is pretty terrible. We could do this any time there is a bug and some tests fail.

The next epoch thing is inevitable because at the epoch boundary crossing for some nodes the root-bank might still be on the previous epoch.
Can you look more into the failing tests and figure out what is going on and find a better approach.

Reply:
Points taken. Thought it might be okay because this code is temporary, but of course we can add wait for one epoch in test as well. I'll see how small the epoch can be so the test doesn't wait for too long.